client/allocrunner/taskrunner/task_runner.go

```diff
@@ -487,7 +487,6 @@ MAIN:
 		// Run the task
 		if err := tr.runDriver(); err != nil {
 			tr.logger.Error("running driver failed", "error", err)
-			tr.EmitEvent(structs.NewTaskEvent(structs.TaskDriverFailure).SetDriverError(err))
```

Comment:
Can you add a comment to `tr.runDriver` that it emits a appropriate task event on error

Reply:
:+1: